app/controllers/write_off_acts_controller.rb

```diff
@@ -3,23 +3,17 @@
 class WriteOffActsController < ApplicationController
   def index
     authorize! :read, WriteOffAct
-    @consignments = Consignment.all
-    @write_off_acts = WriteOffAct.all
-    respond_to do |format|
-      format.html
-      format.json do
-        render json: @write_off_acts.to_json(include: { consignment: { only: %i[bundle_seria
-                                                                                bundle_number] } })
-      end
-    end
+    consignments = Consignment.all
+    write_off_acts = WriteOffAct.all
+    @serialized_write_off_acts = ActiveModelSerializers::SerializableResource.new(write_off_acts).to_json
```

Comment:
maybe better will be using ' new(Consignment.all) '  instead creating new vars

Reply:
We should select consignments and write off acts for these vars. We forgot to change this logic)